modules/highlight/highlight_test.go

```diff
@@ -53,8 +53,6 @@ steps:
 				`</span></span><span class="line"><span class="cl"><span class="w">	</span>- <span class="l">go build -v</span>`,
 				`</span></span><span class="line"><span class="cl"><span class="w">	</span>- <span class="l">go test -v -race -coverprofile=coverage.txt -covermode=atomic</span><span class="w">
 </span></span></span>`,
-				`<span class="w">
-</span>`,
```

Comment:
Should we also add a test where we test for the behavior if there is a newline, and `HIDE_FINAL_NEWLINE = false`?

Reply:
I'm not sure whether these kind of tests should be extended because the output of `go test` is a major pain to work with in case of string arrays. It took me like 10 minutes to figure out what to change to get the test to pass.

Maybe if it were to test a single multiline detented string, it would be easier to work with.

Reply:
Some test improvments in https://github.com/go-gitea/gitea/pull/19798, so I will  revisit here after that one is merged.